public/app/plugins/panel/datagrid/components/AddColumn.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+
+import { SimpleInput } from './SimpleInput';
+
+interface Props {
```

Comment:
I always like naming component props more verbose, in case it ever needs to be exported for testing or referencing in another component to minimize confusion, like `Props ` -> `AddColumnProps `

Reply:
I agree. fixed